protocol/messenger_config.go

```diff
@@ -44,6 +44,7 @@ type MessengerSignalsHandler interface {
 	HistoryArchivesSeeding(communityID string)
 	HistoryArchivesUnseeded(communityID string)
 	HistoryArchiveDownloaded(communityID string, from int, to int)
+	StatusUpdatesTimedOut(statusUpdates interface{})
```

Comment:
Why you use an empty interface instead of a concrete type (UserStatus) ?

Reply:
Replaced with `*[]protocol.UserStatus`